### PR TITLE
add japanese lang test

### PR DESCRIPTION
**Pull request by miurahr**
Signed-off-by: Hiroshi Miura <miurahr@linux.com>

### CODE REVIEW

File: translate/lang/test_ja.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    language = factory.getlanguage('ja')
+    sentences = language.sentences(u"")
+    assert sentences == []
+
+    sentences = language.sentences(u"明日は、明日の風が吹く。吾輩は猫である。\n")
+    assert sentences == [u"明日は、明日の風が吹く。", u"吾輩は猫である。"]
+    sentences = language.sentences(u"頑張れ！甲子園に行きたいか？")
+    assert sentences == [u"頑張れ！", u"甲子園に行きたいか？"]
+
+
```
Comment (unho) [inline translate/lang/test_ja.py]: Please remove these blank lines at end of file.

Comment (miurahr) [inline translate/lang/test_ja.py]: Thanks, done and re-push.

**Comment by jleclanche** [comment]
gtm. @dwaynebailey ?

**Comment by unho** [comment]
gtm

**Comment by unho** [comment]
@miurahr Merged. Thanks for your contribution.

**Pull request merged**